src/scss/3_components/content/_links.scss

```diff
@@ -20,8 +20,7 @@ a {
     background: $link;
     border-radius: 0;
     border: 3px solid transparent;
-    text-decoration: underline;
-    text-underline-offset: .25em;
+    text-decoration: none;
```

Comment:
Does this not remove the underline from all buttons? Is that what we want?

Reply:
Checked with @David-Corbett and it is all buttons.